### ISSUE TITLE
Use the new Server Manager image for the proof tests

### ISSUE THREAD

**Issue by guidanoli** [cartesi/rollups-contracts#246]
## 📚 Context

We are waiting on the Machine Reference Implementation Unit to release a new version of the Server Manager which fully supports the Output Unification feature. Then, we need to update things on our side to use the new image.

**Comment by pedroargento** [comment]
Superseded by #265
